### DISCUSSION TITLE
Welcome to openaudible Discussions!

### DISCUSSION THREAD

**Discussion by openaudible** [openaudible/openaudible#501] [General]
## 👋 Welcome!
  We’re testing Discussions as a place to connect with users of OpenAudible. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Madkow007** [comment]
What are the default directories in OpenAudible ver 2.3.5?
I tried to move a directory to a different hard drive and managed to lose all my files in OpenAudible. 
I know where the axx and mp3 files are, but can not get OpenAudible to see them again.

**Reply by openaudible** [reply]
We added a "diagnostics" menu item for the next release that shows your directories and other info. But you can also find them in the "directories.json" file in the OpenAudible settings dir. (Available from the Preferences, lower left button.) Hope that helps.

**Reply by Madkow007** [reply]
sorry to drag this on...dumb truck driver and audio books are what gets me thru...Is this what this should look like for default directories?
![image](https://user-images.githubusercontent.com/76662718/103323838-4274a080-4a0a-11eb-9318-5b2797b622da.png)
I'm not sure what i did, but the program used to say I had 6000 + hrs of downloaded books and over 500 books left to download

**Reply by Madkow007** [reply]
if I uninstall and reinstall, will that take everything back to default setting? I have not tried to do a full sync because I'm afraid the above directories are wrong

**Reply by openaudible** [reply]
Yeah, those are messed up. Go to preferences, click on the "Show" Preferences at bottom of page. Quit OpenAudible. Delete "Directories.json" file and restart OpenAudible.  That will reset them to the defaults. (Reinstalling does not revert your settings.)

**Reply by Madkow007** [reply]
Thank you...that did the trick. Now to figure how to move everything over to a diff drive without making a mess of things again lol
The info on the web page is somewhat vague.

**Comment by mdl054** [comment]
I'm wondering if Openaudible grabs the highest quality source available? May be a stupid question but in the official audible player when I download the audiobooks to my phone it asks if I want the standard quality or high quality versions. I always grab and very much prefer the high quality versions. I didn't notice an option in the app to change download quality so I assume it either always grabs standard or best available. Thanks!

**Reply by openaudible** [reply]
Yes, we use the best codec available. If you convert to .m4a, you get the best quality copy. .m4b is also available in the next  release, or now in the beta version.

**Comment by bookworm45** [comment]
Feature request: Custom ffmpeg parameters
=============================

I have a playback device that gets very confused by sample rates other than 44.1k, currently I am using OpenAudible to convert once and then ffmpeg a second time for many books that are sampled at 22.05k. I was wondering how easy it would be to add custom arguments to the encoder so this can be done in one step?

Many thanks

**Reply by openaudible** [reply]
Trying to keep the UI as simple as possible. Recommend using m4a or m4b, and converting to mp3 manually if needed.

**Comment by openaudible** [comment]
Quick not to everyone that I totally forgot we enabled this Discussions page. Sorry for the delay responding to the discussions!
